theseus_gui/src/App.vue

```diff
@@ -72,7 +76,9 @@ watch(theme, (newState) => {
           <p>123 mods installed</p>
         </section>
       </div>
-      <RouterView />
+      <Suspense>
+        <RouterView />
+      </Suspense>
```

Comment:
Why was this made suspense?

Reply:
Using await in top level Setup statements requires wrapping the page in Suspense. You'll get errors and warnings about it in the console.

Reply:
ah I see, makes sense